### PR TITLE
update tolerations and nodeSelector of persistence layer

### PR DESCRIPTION
**Pull request by saurabh3460**

### CODE REVIEW

File: app/robot-shop/helm/values.yaml [hunk 1]
```diff
@@ -97,7 +97,15 @@ dispatch:
...
+  tolerations:
+    - key: "persistent"
+      operator: "Equal"
+      value: "true"
+      effect: "NoSchedule"
+  nodeSelector:
+    workload: persistent
+
 
 mysql: {}
```
Comment (Hashfyre) [inline app/robot-shop/helm/values.yaml]: Make `mysql: {}` part of persistent-ng too.

File: app/robot-shop/helm/values.yaml [hunk 2]
```diff
@@ -97,7 +97,15 @@ dispatch:
   nodeSelector:
     workload: app
 
-mongodb: {}
+mongodb:
```
Comment (Hashfyre) [inline app/robot-shop/helm/values.yaml:100]: - change node-group `prod-cluster` -> `app-ng`
- change namespace `prod-robot-shop` -> `robot-shop`
  - `change references to cluster name across the repo, `{makefile, helm values, bash scripts}`
- change cluster name: `prod-eks-cluster` -> `sre-stack`
  - `change references to cluster name across the repo, `{makefile, helm values, bash scripts}`

**Review by Hashfyre — CHANGES_REQUESTED** [review]

**Comment by saurabh3460** [comment]
task:
move these to observability-ng:

```
create-grafana-database-x77gp                          0/1     Completed   0             42m   192.168.173.173   ip-192-168-184-221.us-west-2.compute.internal   <none>           <none>
jaeger-58c79c85cd-fj77z                                1/1     Running     0             39m   192.168.102.26    ip-192-168-118-190.us-west-2.compute.internal   <none>           <none>
kiali-56fc5cf459-wzxww                                 1/1     Running     0             39m   192.168.170.215   ip-192-168-184-221.us-west-2.compute.internal   <none>           <none>
postgresql-psql-0                                      1/1     Running     0             43m   192.168.179.216   ip-192-168-184-221.us-west-2.compute.internal   <none>           <none>
prometheus-prometheus-stack-kube-prom-prometheus-0     2/2     Running     0             41m   192.168.173.173   ip-192-168-184-221.us-west-2.compute.internal   <none>           <none>
prometheus-stack-kube-prom-operator-c68df5867-7n6fk    1/1     Running     0             41m   192.168.110.205   ip-192-168-118-190.us-west-2.compute.internal   <none>           <none>
prometheus-stack-kube-state-metrics-5fc6bbc456-75w65   1/1     Running     0             41m   192.168.161.98    ip-192-168-184-221.us-west-2.compute.internal   <none>           <none>
yace-57f44b74d8-8vjjf                                  1/1     Running     0             39m   192.168.108.167   ip-192-168-118-190.us-west-2.compute.internal   <none>           <none>

```

**Review by Hashfyre — APPROVED** [review]
Should have better code quality going ahead. If DRY is being followed in a repo, it should be followed everywhere.

@saurabh3460 you need to put more thought into the code you write instead of just delivering what is being asked. Follow design / coding guidelines that have been shared with you once across the entire codebase going ahead.

**Pull request merged**